spectral_cube/io/casa_image.py

```diff
@@ -105,6 +105,9 @@ def load_casa_image(filename, skipdata=False, memmap=True,
     elif 'beams' in beam_:
         bdict = beam_['beams']
 
+        # NOTE: temp to check failing test
+        print(desc['_keywords_']['coords'])
```

Comment:
just commenting as a reminder since this is a temp check

Reply:
thanks. still trying to figure out why the dict structure seems to have changed.